twitter_api/main.py

```diff
@@ -16,6 +21,158 @@ def before_request():
 
 
 # implement your views here
+def query_db(query, args=(), one=False):
+    cur = g.db.execute(query, args)
+    rv = [dict((cur.description[idx][0], value)
+               for idx, value in enumerate(row)) for row in cur.fetchall()]
+    return (rv[0] if rv else None) if one else rv
```

Comment:
better to have extra lines unless you're playing code golf imho. two `if/else` statements in the same line is a little hard to read and understand. took me a while.